client/apps/game/src/ui/modules/settings/settings.tsx

```
@@ -92,6 +138,35 @@ export const SettingsWindow = () => {
+        );
+      }
+      if (platform.os?.family === "OS X") {
+        setEternumLoaderDownloadLinks(
+          assets.filter((asset: any) => asset.name.endsWith(".dmg")).map((asset: any) => asset.browser_download_url),
+        );
+      }
+      if (platform.os?.family === "Linux") {
+        setEternumLoaderDownloadLinks([
+          ...assets
+            .filter((asset: any) => asset.name.endsWith(".deb"))
+            ?.map((asset: any) => asset.browser_download_url),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)